test/python/transpiler/test_litinski_transformation.py

```diff
@@ -0,0 +1,98 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test Litinski transformation pass"""
+
+from ddt import ddt, data
+
+from qiskit.circuit import QuantumCircuit, Parameter
+from qiskit.circuit.library import QFTGate
+from qiskit.compiler import transpile
+from qiskit.transpiler.passes import LitinskiTransformation
+from qiskit.quantum_info import Operator
+from test import QiskitTestCase  # pylint: disable=wrong-import-order
+
+
+@ddt
+class TestLitinskiTransformation(QiskitTestCase):
+    """Test the Litinski Transformation pass."""
+
```

Comment:
would be nice to see some tests using https://quantum.cloud.ibm.com/docs/en/api/qiskit/qiskit.quantum_info.get_clifford_gate_names or https://quantum.cloud.ibm.com/docs/en/api/qiskit/circuit_random#random_clifford_circuit (and a fixed seed) to see that we can handle all clifford gates.

Reply:
This is a good idea: I will extend the pass to support the missing standard clifford gates (e.g., ECR) and add tests covering all Clifford gates.